packages/scoutgame/src/builders/approveBuilder.ts

```diff
@@ -40,4 +50,18 @@ export async function approveBuilder({ builderId, season = currentSeason }: { bu
       builderStatus: 'approved'
     }
   });
+
+  if (scout.email) {
+    await sendEmailTemplate({
+      to: { email: scout.email, userId: scout.id },
+      subject: 'Welcome to Scout Game, Builder! 🎉',
+      template: 'Builder Approved',
+      templateVariables: {
+        builder_name: scout.displayName,
+        builder_card_image: scout.builderNfts[0].imageUrl,
```

Comment:
What happens if the image does not exist?

Reply:
an approved builder is guaranteed to have an image url